lib/realtime/supervisor.ex

```diff
@@ -14,6 +14,7 @@ defmodule Realtime.Supervisor do
       {Registry, keys: :duplicate, name: registry_name()},
       {Realtime.BlockWaiverStore, name: Realtime.BlockWaiverStore.default_name()},
       {Realtime.Server, name: Realtime.Server.default_name()},
+      {Realtime.TrainVehiclesPubSub, name: Realtime.TrainVehiclesPubSub.default_name()},
```

Comment:
The data is coming from a separate part of the tree, but it feels to me like what this module is doing is very specific to the realtime functionality supporting client channels. It manages a data store and a registry for channels—it's very equivalent to `Vehicles` or `DataStatusPubSub`.

Reply:
If I'm following right I think this would be another benefit of the decoupling - this has no dependence on the train_vehicle/api streaming stuff, so there's no need to restart this if that restarts.

Reply:
`TrainVehiclesPubSub` has a lot more dependence on the train_vehicle/api streaming than it does on `Server` or `DataStatusPubSub`.